ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/js/dropdownview.js

```diff
@@ -78,11 +78,25 @@
             return String(value) === optionValue;
         }
 
+        #clearValue = function() {
+            [...this.widget].forEach((option) => {
+                if(option.selected) {
+                    option.removeAttribute('selected');
+                }
+            });
+            // selecting the dummy emptyValue option
+            this.widget.selectedIndex = 0;
```

Comment:
selectedIndex should be -1 in case no option is selected.

Reply:
Done